README.md

```diff
@@ -84,6 +84,3 @@ Open the http://localhost:8080 on your browser.
 For any problems, comments, or feedback please create an issue [here on GitHub](https://github.com/Reckue/post-api/issues).
 <br>
```

Comment:
Add license item here without the text `Game of Life`.

Example:
```
Reckue is released under the [MIT license](https://en.wikipedia.org/wiki/MIT_License).
```

Reply:
okay, got it. This issue title confused me.